envelopes/lib/envelopes/current_user.rb

```diff
@@ -0,0 +1,19 @@
+module Envelopes
+    class CurrentUser
+        USER_FILES_PATH = "#{__dir__}/../../users"
```

Comment:
It might be nice to have something in the top level scope where that has an absolute path to the files, and then you can use that to build a path to each file inside of the persistence directory.

```
module Envelopes
  PERSISTENCE_FILES = "#{__dir__}/../whererver_they_are"
end

module Envelopes
  class CurrentUser
    USER_FILES_PATH = "#{PERSISTENCE_FILES}/users"
  end
end
```

Or something like that.
That way the absolute path only lives in one place.

Reply:
Oh yeah.  That is a good thought.  I was thinking that in this case, since there are only two places (and only two directories) it might introduce more complexity.  Maybe down the line, if I use this path more often, I will create a constant to hold the path.  Thanks!!

Reply:
word 👍